backend/src/main/java/com/hyooun/wilson/review/domain/Review.java

```diff
@@ -0,0 +1,111 @@
+package com.hyooun.wilson.review.domain;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Value;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import com.hyooun.wilson.project.domain.Project;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class Review {
+
+	private final ReviewId id;
+
+	private final Project.ProjectId projectId;
+
+	private final String title;
+
+	private final String body;
+
+	private final Reviewer reviewer;
+
+	private final List<Change> changes;
+
+	private ReviewResult reviewResult;
+
+	private ReviewStatus status;
+
+	private final LocalDateTime requestedAt;
+
+	private LocalDateTime completedAt;
+
+	public static Review withoutId(Project.ProjectId projectId,
+		String title,
+		String body,
+		Reviewer reviewer,
+		List<Change> changes) {
+		return new Review(
+			null,
+			projectId,
+			title,
+			body,
+			reviewer,
+			new ArrayList<>(changes),
+			null,
+			ReviewStatus.REQUESTED,
+			LocalDateTime.now(),
+			null
+		);
+	}
+
+	public static Review withId(ReviewId id,
+		Project.ProjectId projectId,
+		String title,
+		String body,
+		Reviewer reviewer,
+		List<Change> changes,
+		ReviewResult result,
+		ReviewStatus status,
+		LocalDateTime requestedAt,
+		LocalDateTime completedAt) {
+		return new Review(id, projectId, title, body, reviewer, changes, result, status, requestedAt, completedAt);
+	}
+
+	public Optional<ReviewId> getId() {
+		return Optional.ofNullable(this.id);
+	}
+
+	public void startGeneration() {
+		if (this.status != ReviewStatus.REQUESTED) {
+			throw new IllegalStateException("리뷰는 REQUESTED 상태에서만 생성 시작 가능");
+		}
+		this.status = ReviewStatus.GENERATING;
+	}
+
+	public void completeReview(ReviewResult result) {
+		if (this.status != ReviewStatus.GENERATING) {
+			throw new IllegalStateException("리뷰는 GENERATING 상태에서만 완료 가능");
+		}
+		this.reviewResult = result;
+		this.status = ReviewStatus.COMPLETED;
+		this.completedAt = LocalDateTime.now();
+	}
+
+	public boolean isCompleted() {
```

Comment:
Review 클래스에 ReviewId VO 추가 필요.

```suggestion
public static class ReviewId { Long value; }
```